src/ServiceHelper.php

```diff
@@ -93,9 +93,14 @@ public static function getServiceIdFromNode(Expr $node, Scope $scope): ?string
 
     private function createKernel(): void
     {
+        putenv('DB_HOST=');
```

Comment:
Is it necessary to unset these variables or did you just do it because they are not used?

Reply:
Its actually not *unsetting* them, its setting them because they are required by the test configuration https://github.com/contao/contao/blob/master/core-bundle/tests/Functional/app/config/config_test.yml#L35-L38